src/pages/integrations/coinbase/coinbase.html

```diff
@@ -14,77 +14,26 @@
     <expandable-header [scrollArea]="scrollArea" [fadeFactor]="5" [disableFade]="true" #expandableHeader>
       <ion-toolbar class="wide-header__title">
         <expandable-header-primary>
-          <img margin-left src="assets/img/coinbase/coinbase-logo.png" width="150">
-          <ion-spinner *ngIf="accessToken && loading"></ion-spinner>
+          <img src="assets/img/coinbase/coinbase-logo.png" width="150">
         </expandable-header-primary>
       </ion-toolbar>
     </expandable-header>
 
-    <div *ngIf="!accessToken" class="center-header">
+    <div class="center-header">
       <div padding text-center>
-        <div *ngIf="!showOauthForm">Coinbase's exchange service is available in US, and can take 3-5 days to buy or sell bitcoin.</div>
+        <div *ngIf="!showOauthForm">Manage your Coinbase's accounts, check balance, deposit and withdrawal funds between wallets.</div>
```

Comment:
I would change this sentence to "Manage your Coinbase account, check balances, deposit and withdraw funds between wallets."

Reply:
Fixed!